StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -146,6 +149,26 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
             XCTAssertEqual(form.getAllSubElements().count, 1)
         }
     }
+
+    func testAlipayConfirmFlows() async throws {
+        try await _testConfirm(intentKinds: [.paymentIntent],
+                               currency: "USD",
+                               paymentMethodType: .dynamic("alipay"),
```

Comment:
such a code smell that we do `.dynamic("alipay")`. What does it mean that this is dynamic?? I don't think we can change anything in this PR, just pointing it out.